.github/workflows/conformance-clustermesh.yaml

```diff
@@ -206,7 +206,7 @@ jobs:
         run: |
 
           CILIUM_INSTALL_DEFAULTS="${{ steps.default_vars.outputs.cilium_install_defaults }} \
-            --helm-set=kubeProxyReplacement=${{ matrix.kube-proxy == 'none' }} \
+            --helm-set-string=kubeProxyReplacement=${{ matrix.kube-proxy == 'none' }} \
```

Comment:
Why are changes to `set-string` necessary? Previously we were accepting `kubeProxyReplacement` as either a bool or a string, thanks to `{{- $stringValueKPR := (toString .Values.kubeProxyReplacement) -}}`.

I think we should continue to accept plain booleans. I think only accepting `"true"` and `"false"` (string) instead of `true`/`false` (boolean) will lead to a bad user experience.

Reply:
This is to make it work with current cilium-cli version, because the CLI always overrides it with strict. Using --set-string is a workaround in that case.

Probably need to wait for cilium-cli PR to land in first https://github.com/cilium/cilium-cli/pull/2385